duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/inputscreen/ui/viewmodel/InputScreenViewModel.kt

```diff
@@ -340,6 +352,16 @@ class InputScreenViewModel @AssistedInject constructor(
         }
     }
 
+    fun toggleForceWebSearch() {
+        viewModelScope.launch {
+            _forceWebSearchState.update {
+                it.copy(
+                    shouldForceWebSearch = !it.shouldForceWebSearch,
```

Comment:
Wondering if this should persist across Input Screen launches? We can check and follow up separately.

Reply:
Yeah, I’m not sure if it needs to persist, can fix in a follow-up if needed.